src/main/java/randoop/reflection/OperationModel.java

```diff
@@ -8,10 +8,11 @@
 import java.util.List;
 import java.util.Set;
 
-import randoop.contract.EqualsHashcode;
+import randoop.contract.EqualsTransitive;
 import randoop.contract.EqualsReflexive;
 import randoop.contract.EqualsSymmetric;
 import randoop.contract.EqualsToNullRetFalse;
+import randoop.contract.EqualsHashcode;
```

Comment:
Why has this been moved later?  I would keep them in their current order to minimize diffs.